controller/tap/server.go

```diff
@@ -426,18 +432,53 @@ func (s *server) translateEvent(orig *proxy.TapEvent) *public.TapEvent {
 	return ev
 }
 
+func newHTTPServer(k8sAPI *k8s.API, controllerNamespace string) *http.Server {
+	var (
+		aggregationLayerConfigMapName = "extension-apiserver-authentication"
+	)
+
+	cm, err := k8sAPI.Client.CoreV1().
+		ConfigMaps("kube-system").
+		Get(aggregationLayerConfigMapName, metav1.GetOptions{})
+
+	if err != nil {
+		log.Fatalf("Failed to load [%s] config: %s", aggregationLayerConfigMapName, err)
```

Comment:
Good point. I think if we can't create this server, for now, we want to fail silently. When the second PR comes along, this would mean that there may be an issue with the cluster and we want the pod to restart. I will include tests to this to capture the initial behavior.